data/json/monsterdrops/zombie_default.json

```diff
@@ -9,6 +9,7 @@
       { "group": "common_gloves", "damage": [ 1, 4 ], "prob": 20 },
       { "group": "hatstore_hats", "damage": [ 1, 4 ], "prob": 20 },
       { "group": "scarfs_unisex", "damage": [ 1, 4 ], "prob": 20 },
+      { "group": "clothing_glasses", "damage": [ 1, 4 ], "prob": 10 },
       { "group": "accesories_personal_unisex", "damage": [ 1, 4 ], "prob": 15 },
```

Comment:
`accesories_personal_unisex` item group already contains `clothing_glasses` item group with 10% chance to spawn, so with this your addition sometimes zombie will spawn with two pairs of glasses.

Reply:
huh, the glasses must be incredibly rare in that case, because they don't appear in real gameplay, I'll close this and edit that instead, my bad.